src/Microsoft.IdentityModel.Tokens.Saml/SamlTokenUtilities.cs

```diff
@@ -102,5 +104,60 @@ internal static IEnumerable<SecurityKey> GetKeysForTokenSignatureValidation(stri
             }
             return null;
         }
+
+        /// <summary>
+        /// Creates claims from  a dictionary.
+        /// </summary>
+        /// <param name="claimsCollection"> A dictionary that represents a set of claims.</param>
+        /// <returns> A collection of Claim objects created from the dictionary.</returns>
+        internal static IEnumerable<Claim> CreateClaimsFromDictionary(IDictionary<string, object> claimsCollection)
+        {
+            List<Claim> claims = null;
+            object value;
```

Comment:
nit: this can be inlined with the out parameter

Reply:
Done.